keps/sig-node/4009-add-cdi-devices-to-device-plugin-api/README.md

```diff
@@ -148,14 +148,14 @@ However, these cases will be added in the existing integration tests:
 
 ##### e2e tests
 
-These cases will be added in the existing `e2e_node` tests:
-  - Device Plugin works with CDI devices
+This test case has been added to the existing `e2e_node` tests:
+  - DevicePlugin can make a CDI device accessible in a container
 
-Links to tests:
-- TBD: Will fill-in by code freeze
+Links to test grid:
+- https://testgrid.k8s.io/sig-node-containerd#e2e-cos-device-plugin-gpu
 
 Links to k8s-triage for tests:
-- TBD: Will fill-in by code freeze
+- https://storage.googleapis.com/k8s-triage/index.html?test=DevicePlugin
```

Comment:
and totally flake https://storage.googleapis.com/k8s-triage/index.html?test=DevicePlugin